enterprise/frontend/src/metabase-enterprise/auth/components/SettingsJWTForm/SettingsJWTForm.unit.spec.tsx

```diff
@@ -22,7 +22,7 @@ const elements = [
     is_env_setting: false,
     env_name: "MB_JWT_ENABLED",
     description: "Is JWT authentication configured and enabled?",
-    default: false,
+    // default: false,
```

Comment:
We should probably remove these commented values

Reply:
fbf0f6a408be7b7780e1b583da7c51f347d8a4f4